src/blob_storage.h

```diff
@@ -65,6 +66,7 @@ class BlobStorage {
   void MarkAllFilesForGC() {
     MutexLock l(&mutex_);
     for (auto& file : files_) {
+      file.second->set_gc_mark(true);
```

Comment:
hmm, sorry for notice this late, but since we have the `kDbRestart` state, why we need an extra boolean flag `gc_mark`?

Reply:
because FileState will set to normal when FileStateTraist received FileEvent::kDbRestart, we cannot know a normal file is generated by restart or compaction

Reply:
Got it. I misunderstand the code.